<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BanqueMainWindow</class>
 <widget class="QMainWindow" name="BanqueMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BanqueMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonQuitter">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>490</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>601</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Connexion</string>
    </property>
    <widget class="QPushButton" name="pushButtonConnect">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>70</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Connexion a la bdd</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>271</width>
       <height>124</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="labelIP">
          <property name="text">
           <string>Adresse IP</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditIP">
          <property name="text">
           <string>172.18.58.14</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="labelName">
          <property name="text">
           <string>Nom de la bdd</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditName">
          <property name="text">
           <string>banque</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="labelLogin">
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditLogin">
          <property name="text">
           <string>snir</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="labelPassword">
          <property name="text">
           <string>Mot de passe</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditPassword">
          <property name="text">
           <string>snir</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>601</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Affichage infos comptes</string>
    </property>
    <widget class="QPushButton" name="pushButtonInfoComplete">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>351</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Afficher les informations de tous les comptes</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidgetInfos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>581</width>
       <height>192</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>144</number>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Nouvelle colonne</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nom</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Prenom</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Solde</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>BanqueMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
